src/static/js/libs/gamepad.js

```diff
@@ -562,12 +562,12 @@
     if (this.gamepads.length > 0) {
       this._fire(Gamepad.Event.TICK, this.gamepads);
     }
-    //window.requestAnimationFrame(function() {
-    //	self._update();
-    //});
-    setTimeout(function () {
-      self._update();
-    }, 100);
+    window.requestAnimationFrame(function() {
```

Comment:
Will run performance tests, as is. This was in response to @spiderkeys suggestion to move the communication rate to the ROV from 10 Hz to 60 Hz. Those changes are not in the pipeline yet. 

It is worth noting, as well, that this does not only affect positional updates to the ROV.